frontend/src/components/Calendar/EventModal.js

```diff
@@ -4,6 +4,7 @@ import api from "../../utils/api";
 import Modal from "react-modal";
 import {checkHubPrivileges, checkAuthorPrivileges} from "../../utils/fetchPrivileges";
 import { getHubUrl } from "@/utils/url-segments";
+import { formatDate } from "@/utils/url-segments";
```

Comment:
The formatDate function doesn't seem to be in url-segments. I think you meant to import formatDate from the datatime-conversion file that you just made. The current import results in the following runtime error:

![image](https://github.com/user-attachments/assets/ebf54c41-bbd8-4c72-b978-4342e62870d7)